script/Arcueid/Day 5/05_00_ARC05_8.txt

```diff
@@ -510,12 +510,12 @@ And then―――
 [sha:5a34da987dbe2ad8e37cd710badd8724996c6c5a]{
 -- Page 551, Offset 10080.
 -- <彼|か>の領土はこの星の一部でありながら、“生きるに<能|あた>わず”と除外された生命圏。
-Although this domain remains of the planet, all the same is it a biosphere foreclosed... in which none may live...
+Although this domain is part of the planet, all the same is it a biosphere foreclosed, in which none can live...
```

Comment:
it is a biosphere foreclosed all the same,

Reply:
I feel like the follow up doesn't flow as well then. Is that just me?

> Although this domain is part of the planet, it is a biosphere foreclosed all the same, in which none can live...

Reply:
Yeah I think what you already have here is better